Remove FLAG_LINK_INTR because otherwiese usbnet.c will set the curruent link
state to down during initialization. Currently we do not support link-state
changes for SMSC95XX because the driver uses mdio/phy/irqchip infrastructure
that is hard to enable. Therefore, the driver must be started with NIC whose
link is up.

diff --git a/drivers/net/usb/smsc95xx.c b/drivers/net/usb/smsc95xx.c
index 32d2c60..bd7f0be 100644
--- a/drivers/net/usb/smsc95xx.c
+++ b/drivers/net/usb/smsc95xx.c
@@ -2002,7 +2002,7 @@ static const struct driver_info smsc95xx_info = {
 	.tx_fixup	= smsc95xx_tx_fixup,
 	.status		= smsc95xx_status,
 	.manage_power	= smsc95xx_manage_power,
-	.flags		= FLAG_ETHER | FLAG_SEND_ZLP | FLAG_LINK_INTR,
+	.flags		= FLAG_ETHER | FLAG_SEND_ZLP,
 };
 
 static const struct usb_device_id products[] = {
